.github/workflows/create-quantic-package.yml

```diff
@@ -57,11 +57,13 @@ jobs:
           npm run setup
           cd packages/quantic
           npm run build
-      - name: Create jwt key file
-        run: echo "$SFDX_AUTH_JWT_KEY" > server.key
       - name: Connect to DevHub organization
-        run: sfdx force:auth:jwt:grant --clientid $SFDX_AUTH_JWT_CLIENT_ID --jwtkeyfile server.key --username $SFDX_AUTH_JWT_USERNAME --setdefaultdevhubusername
+        working-directory: ./packages/quantic
+        run: |
+          echo $SFDX_AUTH_JWT_KEY > server.key
+          sfdx force:auth:jwt:grant --clientid $SFDX_AUTH_JWT_CLIENT_ID --jwtkeyfile server.key --username $SFDX_AUTH_JWT_USERNAME --setdefaultdevhubusername
       - name: Create package version
+        working-directory: ./packages/quantic
```

Comment:
do you need this to execute inside a directory? does L58 still work then?

Reply:
I don't think L58 will be affected by the `working-directory` param since it's in a different step. I added it here because I realized that the command for authorizing the DevHub is listed in the quantic directory's README so I thought we needed to run it from there. Is that not the case?

Reply:
No i was just curious because cd and working-directory seem to do something similar so I just wanted to make sure cd works as well. Just asking questions so we limit the number of PRs :P

Reply:
true! But yes cd worked in the previous PR